config/locales/en/views.yml

```diff
@@ -303,18 +303,27 @@ en:
       yes_cancel: Yes, I want to cancel this visit
```

Comment:
Any ideas on how to manage the translations? We have most of this text localised from the old completed step (which I haven't removed in this PR)